docs/swagger.yml

```diff
@@ -321,6 +346,52 @@ paths:
           description: "Scan job not found"
 
 definitions:
+  OSFact:
```

Comment:
SystemFact would have been better, as we will quickly add non-os related facts here.